shared/more/dumb-map.js

```diff
@@ -0,0 +1,24 @@
+import CommonMap from '../common-adapters/dumb.desktop'
```

Comment:
Perhaps I should rename this file (and its associated variables) to `dumb-component-map.js`